src/lib/annotations/Annotator.scss

```diff
@@ -550,6 +548,22 @@ $tablet: 'max-width: 768px';
     }
 }
 
+.bp-annotation-draw .textLayer {
+    pointer-events: none;
+
+    > div {
+        pointer-events: auto;
```

Comment:
@tonyjin The trick here is to remove pointer events on the overall blocking div but re-enable them on the text layer divs so that both behaviors an coexist.